font-patcher

```diff
@@ -293,6 +293,11 @@ class font_patcher:
         # some fonts have inaccurate 'SubFamily', if it is Regular let us trust the filename more:
         if subFamily == "Regular":
             subFamily = fallbackStyle
+
+        # This is meant to cover the case where the SubFmaily is "Italic" and the filename is *-BoldItalic.
+        if  len(subFamily) < len(fallbackStyle):
```

Comment:
Is it a simple length comparison a sufficient check here? Instead of checking sub family is italic _and_ fallbackStyle contains italic _and_ longer in length?

Honestly asking as it's been a while since I worked on the patcher :smile:

Reply:
You know, I did fix the typo...